net/frr/src/opnsense/service/templates/OPNsense/Quagga/ripd.conf

```diff
@@ -0,0 +1,34 @@
+{% if helpers.exists('OPNsense.quagga.rip.enabled') and OPNsense.quagga.rip.enabled == '1' %}
+{% from 'OPNsense/Macros/interface.macro' import physical_interface %}
+!
+! Zebra configuration saved from vty
+!   2017/03/26 22:40:16
+!
+{% if helpers.exists('OPNsense.quagga.general') %}
+{%   if helpers.exists('OPNsense.quagga.general.enablelogfile') and OPNsense.quagga.general.enablelogfile == '1' %}
+log file /var/log/frr.log {{ OPNsense.quagga.general.logfilelevel }}
+{%   endif %}
+{%   if helpers.exists('OPNsense.quagga.general.enablesyslog') and OPNsense.quagga.general.enablesyslog == '1' %}
+log syslog {{ OPNsense.quagga.general.sysloglevel }}
+{%   endif %}
+{% endif %}
+!
+router rip
```

Comment:
the old one also had:

```
router zebra
 no redistribute rip
!
```

Reply:
@fichtner this is an error in FRR. I am not sure about Quagga but nobody did ever complain about it so it might not raise an error there.

Reply:
could this be reported to FRR? should we keep the code as a comment?

both up to you :)

Reply:
I don't thing this should be included anyway but my first version came from generated configuration files by quagga where it was included.